.github/workflows/canary-release.yml

```diff
@@ -4,26 +4,33 @@ on:
   workflow_dispatch:
     inputs:
       label:
-        description: 'Pick a label for the release, to be included in the version number — e.g. "fix-button-focus" will released as "@kaizen/button@1.0.0-fix-button-focus-20230719002814"'
+        description: 'Pick a label for the release, to be included in the version number — e.g. "fix-button-focus" will released as "@kaizen/button@0.0.0-canary-fix-button-focus-20230719002814"'
         required: true
         type: string
 
 jobs:
   release:
     runs-on: ubuntu-latest
+    outputs:
+      published: ${{ steps.changesets.outputs.published }}
```

Comment:
This won't work because `changesets` isn't a step in this job. We'll try use the pushed tags in a different job to launch the notification

Reply:
Right, I suppose this isn't being used right now so it can be removed